### ISSUE TITLE
5$: missing comma between object literal entries should report error

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#912]
```javascript
    let newTokens = [
        {...headingOpenToken, attrs: [...headingOpenToken.attrs ?? [], ['id', currentVersionID]]},
        {
          type: 'link_open',
          tag: 'a',
          attrs: [['href', `#${currentVersionID}`]],
        },
        ...textTokens    // <--- bug (but quick-lint-js reports nothing)
        {
          type: 'link_close',
          tag: 'a',
        },
        headingCloseToken,
      ];
```
